core/tests/protractor_desktop/playVoiceovers.js

```diff
@@ -54,8 +54,15 @@ describe('Voiceover player', function() {
     await workflow.createExploration();
     await explorationEditorMainTab.exitTutorial();
     await explorationEditorMainTab.setStateName('First');
-    await explorationEditorMainTab.setContent(await forms.toRichText(
-      'This is the first card.'));
+    await explorationEditorMainTab.setContent(async function(richTextEditor) {
+      await richTextEditor.appendPlainText('This is the first card.');
+      await richTextEditor.addRteComponent('Math', 'abc');
+    });
+    await explorationEditorMainTab.expectContentToMatch(
+      async function(richTextChecker) {
+        await richTextChecker.readPlainText('This is the first card.');
+        await richTextChecker.readRteComponent('Math', 'abc');
+      });
```

Comment:
This seems pretty out of place for the voiceovers suite. Can you put it in a new suite? Since this is a new test, that'll also help isolate the damage if it turns out to be flaky

Reply:
Also, since the e2e test flakiness has gotten really bad recently, we are now asking everyone to make sure their e2e tests pass consistently and add tests in separate suites. This process is detailed in the [end-to-end test wiki page](https://github.com/oppia/oppia/wiki/End-to-End-Tests). In your case, since this PR modifies both utility files and introduces a new test, please post screenshots showing:

* 5 consecutive passes of your new test suite and
* 3 consecutive passes of all e2e tests.

Note that unrelated flakes don't count toward the required number of passes, but nor do they count against the "consecutive" requirement.

Reply:
done, moved this test to a new suite.
Note: I'll be changing how math RTE components are rendered in the future, So I've planned to add more e2e tests after that is finished.

Reply:
> Note that unrelated flakes don't count toward the required number of passes, but nor do they count against the "consecutive" requirement.
Does this mean if an unrelated test flaked I have to re-run everything?

Reply:
Yes, in general it does. However, note that when trying to get 5 consecutive passes of your new test suite, you'd only be re-running your suite. If it's on CircleCI where you have to rerun the whole job, then we can ignore flakes in unrelated suites so long as they don't stop your suite from running.

The rationale behind this is that even if a flake is unrelated, we can't count that as a passing run if the flake stopped your test from running correctly. Does that make sense?